src/Task/Git/Blacklist.php

```diff
@@ -81,6 +84,11 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments->add('-n');
         $arguments->add('--break');
         $arguments->add('--heading');
+
+        if (true === $matchWord) {
```

Comment:
To make the task more readable, you should go for:

```php
$arguments->addOptionalArgument('--word-regexp, $config['match_word']);
```

(1 line instead of 6)

Reply:
Oh yeah, of cause. 
At first i could not figure out how to get the tests to pass with a boolean (i kept getting an error that the default value was incorrect) So i just forgot to remove the if.

I will make that change.

Reply:
This has been committed